src/javascript/Create/CreateNewContentAction/createNewContent.action.jsx

```diff
@@ -4,49 +4,72 @@ import {componentRendererAction, composeActions} from '@jahia/react-material';
 import {reduxAction} from '../../actions/redux.action';
 import {Constants} from '~/ContentEditor.constants';
 import {cmGoto} from '../../ContentManager.redux-actions';
+import {withApolloAction} from './withApolloAction';
+import {from} from 'rxjs';
```

Comment:
Why ?

Reply:
Because the action framework requires to have Observables

Reply:
So sad